### PR TITLE
Quick add to cart: scroll to top of page

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Update quick "add to cart" button on the footer of PDP page (mobile only)

Scroll all the way to the top of the page when clicking on the "add to cart" button

## Justify why they are needed

Before it just scrolled to the main "add to cart" button, which looks a bit weird.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1786** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1786" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1786?utm_source=stack-comment).

File: apps/store/src/components/ProductPage/ScrollToButton/ScrollToButton.tsx [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
+type ScrollToTopButtonProps = {
   type: 'button' | 'submit'
-  targetRef: React.RefObject<HTMLElement>
   children: ReactNode
 }
 
-export const ScrollToButton = ({ children, type, targetRef }: ScrollToButtonProps) => {
-  const handleClick = () => targetRef.current?.scrollIntoView({ behavior: 'smooth' })
+export const ScrollToTopButton = ({ children, type }: ScrollToTopButtonProps) => {
+  const handleClick = () => window.scrollTo({ top: 0 })
```
Comment (gustaveen) [inline apps/store/src/components/ProductPage/ScrollToButton/ScrollToButton.tsx:11]: Could there be an idea to keep the `ref` as optional and scroll to top if no ref is provided?

**Review by gustaveen — APPROVED** [review]
Great!

**Comment by robinandeer** [comment]
@gustaveen we could but pretty easy to add back - don't want to keep features we are not using ;)

**Comment by gustaveen** [comment]
> @gustaveen we could but pretty easy to add back - don't want to keep features we are not using ;)

@robinandeer makes sense! 🙌

**Pull request merged**